debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnectorConfig.java

```diff
@@ -273,6 +274,17 @@ public static SnapshotIsolationMode parse(String value, String defaultValue) {
             .withDescription("The timezone of the server used to correctly shift the commit transaction timestamp on the client side"
                     + "Options include: Any valid Java ZoneId");
 
+    public static final Field SKIP_LSN_TIMESTAMP_QUERY = Field.create(DATABASE_CONFIG_PREFIX + SKIP_LSN_TIMESTAMP_QUERY_CONFIG_NAME)
```

Comment:
The "database" prefix is only meant for pass-through options which are passed on to the JDBC driver. It's not needed here.

Reply:
That was my first intention. I tried without the `database.` prefix, but I realized it was then filtered out at the moment `SqlServerConnection` was created:

```java
        final Configuration jdbcConfig = config.filter(
                x -> !(x.startsWith(DatabaseHistory.CONFIGURATION_FIELD_PREFIX_STRING) || x.equals(HistorizedRelationalDatabaseConnectorConfig.DATABASE_HISTORY.name())))
                .subset("database.", true);
        dataConnection = new SqlServerConnection(jdbcConfig);
```

Maybe I was doing something wrong, but as far as I remember, I had to include the prefix.

Reply:
Would be good to double-check. It shouldn't be needed.